csfieldguide/templates/interactives/training-ground.html

```diff
@@ -0,0 +1,114 @@
+            </div>
+            <div id="quit-buttons" class="col-12 text-right d-none">
+              <button id="button_quit" type="button" class="btn btn-danger m-2">{% trans 'Quit' %}</button>
+            </div>
+            <div id="splash-text-container">
+              <div id="splash-text" class="col-12 d-none"></div>
+              <button id="button_cancel" type="button" class="btn btn-danger btn-lg d-none">{% trans 'STOP' %}</button>
+            </div>
+          </div>
+          <table id="data-table" class="col-5 text-center"></table>
+        </div>
+      </div>
+      <div id="game-parameters">
+        <h3>{% trans 'Game parameters:' %}</h3>
+        <div class="row">
+          <div class="col-12 col-md-4 form-group text-center">
+            <label for="who-starts-select">{% trans 'Nathaniel starts (%):' %}</label>
+            <small id="who-starts-help" class="form-text text-muted">{% trans 'How likely is the AI to start each game?' %}</small>
+            <div id="who-starts-select" class="m-2"></div>
+          </div>
+          <div class="col-12 col-md-4 form-group text-center">
+            <label for="num-sticks-select">{% trans 'Number of sticks:' %}</label>
+            <small id="num-sticks-help" class="form-text text-muted">{% trans 'How many sticks in the pile?' %}</small>
+            <div id="num-sticks-select" class="m-2"></div>
+          </div>
+          <div class="col-12 col-md-4 form-group text-center">
+            <label for="sensitivity-select">{% trans 'AI sensitivity (%):' %}</label>
+            <small id="sensitivity-help" class="form-text text-muted">{% trans 'How much to change after each game.' %}</small>
+            <div id="sensitivity-select" class="m-2"></div>
+          </div>
+          <div class="col-12 col-md-4 form-group text-center">
+            <label for="num-simulations-select">{% trans 'Number of simulations:' %}</label>
+            <small id="num-sims-help" class="form-text text-muted">{% trans 'Nathaniel can practice before we start.' %}</small>
+            <div id="num-simulations-select" class="m-2"></div>
+          </div>
+          <div class="col-12 col-md-4 form-group text-center">
+            <label for="practice-opponent-select">{% trans 'Practice opponent:' %}</label>
+            <small id="practice-opponent-help" class="form-text text-muted">{% trans 'What is Nathaniel&#39;s opponent during simulations?' %}</small>
+            <div id="practice-opponent-select" class="btn-group btn-group-toggle mt-3" data-toggle="buttons">
+              <label class="btn btn-secondary active">
+                <input type="radio" name="options" id="radio_smartbot" autocomplete="off" value="1" checked>{% trans 'Smart bot' %}
+              </label>
+              <label class="btn btn-secondary">
+                <input type="radio" name="options" id="radio_dumbbot" autocomplete="off" value="2">{% trans 'Random bot' %}
```

Comment:
What is random bot? and smart bot etc. The user probably won't know what the implications of selecting these options are. Can we please explain this somewhere? :)